tests/redirects_file_test.go

```diff
@@ -224,46 +142,16 @@ func TestRedirectsFileSupport(t *testing.T) {
 					Status(200).
 					Body(Contains("my index")),
 			},
-			// # This test ensures _redirects is supported only on Web Gateways that use Host header (DNSLink, Subdomain)
-			// test_expect_success "request for http://127.0.0.1:$GWAY_PORT/ipfs/$REDIRECTS_DIR_CID/301-redirect-one returns generic 404 (no custom 404 from _redirects since no origin isolation)" '
-			//
-			//	curl -sD - "http://127.0.0.1:$GWAY_PORT/ipfs/$REDIRECTS_DIR_CID/301-redirect-one" > response &&
-			//	test_should_contain "404 Not Found" response &&
-			//	test_should_not_contain "my 404" response
-			//
-			// '
-			// {
-			// 	// TODO: confirm with lidel: this should be skipped
-			// 	Name: "This test ensures _redirects is supported only on Web Gateways that use Host header (DNSLink, Subdomain)",
-			// 	Hint: `
-			// We expect the request to fail with a 404 (do not use the _redirect), and that 404 should not contain the custom 404 body.
-			// `,
-			// 	Request: Request().
-			// 		URL("http://127.0.0.1:8080/ipfs/{{etag}}/301-redirect-one", redirectDirCID),
-			// 	Response: Expect().
-			// 		Status(404).
-			// 		Body(Not(Contains(custom404.ReadFile()))),
-			// },
```

Comment:
Since we're testing a single configuration with each conformance run, that means running a check if a spec is NOT enabled, right?

We'd have to use a definition like:

```go
if ((!specs.SubdomainGatewayIPFS || !specs.DNSLink) && specs.RedirectsFile) {
	Run(test)
}
```


If that's the case, what about we create a follow-up task for this?
That means:
- thinking about NOTs in the specs filters,
- discussing how to run these tests in Kubo, it might mean having multiple conformance test runs with different configurations.

Reply:
Yes, its tricky – fine to extract this to follow-up task :+1: